### PR TITLE
Add test command

### PR DESCRIPTION
**Pull request by arturdm**

### CODE REVIEW

File: lib/src/command/test_command.dart [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import 'package:args/command_runner.dart';
+import 'package:moronepo/src/project_finder/project.dart';
+import 'package:moronepo/src/project_finder/project_finder.dart';
+
+import 'command_and_arguments_formatter.dart';
+
+class TestCommand extends Command<Null> {
+  String get workingDirectory => _fromGlobalResults("working-directory");
+
+  String get projectName => _fromGlobalResults("project");
```
Comment (WezSieTato) [inline lib/src/command/test_command.dart]: This code and above are duplicated. What do you think about create `MoronepoResults` class that will have these properties and `MoronepoCommand` that will has `moronepoResults` property and other commands will inherit it?

File: test/command/run_command_integration_test.dart [hunk 2]
```diff
@@ -5,14 +5,15 @@ import 'package:test/test.dart';
 
 void main() {
   group("run", () {
-    
     test("should notify if no projects found", () async {
-      final emptyDirectory = "${Directory.current.path}/test/command/test_project/empty_directory";
+      final emptyDirectory =
+          "${Directory.current.path}/test_resources/command/test_project/empty_directory";
```
Comment (WezSieTato) [inline test/command/run_command_integration_test.dart]: Do we need `command` directory? Can we  move `test_project` and `test_project_fo_test` one directory up?

File: analysis_options.yaml [hunk 3]
```diff
@@ -1,6 +1,7 @@
 include: package:pedantic/analysis_options.1.8.0.yaml
 
 analyzer:
+  exclude: [test_resources/**]
```
Comment (WezSieTato) [inline analysis_options.yaml]: Why we don't want to have beautiful test projects :P ?

Comment (arturdm) [reply analysis_options.yaml]: I'll try to check how I can fix the analysis error that occurs there. It seemed like something related to imports but the project actually works if you try to run it from its directory :/.

File: test/command/run_command_integration_test.dart (hunk 2 above)
Comment (arturdm) [reply test/command/run_command_integration_test.dart]: Currently, there's a 1-1 relation of the directory layout in tests and production code if those tests are in `command` directory.

File: test/current_directory.dart [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import 'dart:io';
+
+import 'package:path/path.dart';
+
+final Directory testDirectory = Directory(join(
```
Comment (WezSieTato) [inline test/current_directory.dart]: Filename is weird/misleading for me. What do you think about `test_directory`\`test_directories`?

**Review by WezSieTato — APPROVED** [review]

File: test/current_directory.dart (hunk 4 above)
Comment (arturdm) [reply test/current_directory.dart]: I'll call it directories since `projectDirectory` is the one used and derived from `testDirectory`

**Pull request merged**